scripts/run-script.js

```diff
@@ -67,6 +67,10 @@ require('exec-promise')(args => {
           }),
         }
         return forEach.call([`pre${script}`, script, `post${script}`], script => {
+          if (script === 'test') {
+            if (scripts['integration-test'] !== undefined) script = 'integration-test'
+          }
```

Comment:
I don't understand this change.

Reply:
It's to execute integration tests during unit tests. I think it's the current behavior with ```jest \"^(?!.*\\.integ\\.spec\\.js$)\"```.

Reply:
`test-unit` runs all files called `*.spec.js` except files named `*.integ.spec.js`.

Reply:
Ok my bad. I'll remove that.